safe/definitions/post_processors.py

```diff
@@ -248,6 +313,8 @@ def post_processor_affected_function(**kwargs):
                 ],
             }]
     },
+    # output is described as ordered dict because the order is important
+    # and the postprocessor produce two fields.
```

Comment:
Do you mean that the postprocessor will produce two fields ? I need to check deeper how you could do that